app/services/physical_holdings_markup_builder.rb

```diff
@@ -276,10 +276,21 @@ def self.listify_array(arr)
     arr.join
   end
 
+  def doc_id(holding)
+    host_id = adapter.host_id
```

Comment:
Curious here why this is dependent on adapter host_id and not just the holding information.  I might have expected to see something simpler like `holding.dig("mms_id") || adapter.doc_id` replace this entire method.  It looks to me like you set up for that at line 190 of solr_document.

Reply:
You are right. I think it was because initially I append the mmsid only in the host holdings and later I decided to append it in the record holding as well.  The holding should include all the mms ids; but I will also leave the one you suggested || adapter.doc_id just to be safe